src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/ubuntu/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -46,5 +45,21 @@
     </File>
   </ItemGroup>
 
+  <ItemGroup Condition="'$(__BuildType)' == 'Release'">
+    <ArchitectureSpecificNativeSymbol Include="@(NativeSplittableBinary -> '%(Identity).dbg')"/>
+
+    <AdditionalLibPackageExcludes Include="%2A%2A\%2A.dbg"/>
+    <AdditionalSymbolPackageExcludes Include="%2A%2A\%2A.so"/>
+    <AdditionalSymbolPackageExcludes Include="%2A%2A\%2A.dll"/>
+    <AdditionalSymbolPackageExcludes Include="%2A%2A\sosdocsunix.txt"/>
+    <AdditionalSymbolPackageExcludes Include="%2A%2A\crossgen"/>
+    <ArchitectureSpecificNativeSymbol Include="..\_.pdb"/>
+
+    <File Include="@(ArchitectureSpecificNativeSymbol)">
+      <TargetPath>runtimes/$(PackageTargetRuntime)/native</TargetPath>
+      <IsSymbolFile>true</IsSymbolFile>
```

Comment:
Right, the package gets split up by the path globs in this file plus the ones in buildtools: https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks.Packaging/src/NuGetPack.cs#L19-L34

@gkhanna79 Did you want more details why or is that fine?